src/Microsoft.Framework.Caching.Redis/CachingServiceExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Framework.Caching.Distributed;
+using Microsoft.Framework.Caching.Redis;
+namespace Microsoft.Framework.DependencyInjection
```

Comment:
Code style: can you add an empty line before the namespace? (In other files too)